webapp/settings.py

```diff
@@ -35,7 +35,7 @@
 STATIC_URL = '/static/'
 STATICFILES_FINDERS = ['django_static_root_finder.StaticRootFinder']
 TEMPLATE_DIRS = ['templates']
-ASSET_SERVER_URL = 'https://assets.ubuntu.com/v1/'
+ASSET_SERVER_URL = '//assets.ubuntu.com/v1/'
```

Comment:
Can I request you change this back to https? Non https links on dev breaks the content security policy. :D